app/assets/src/components/utils/links.js

```diff
@@ -21,4 +21,14 @@ const downloadStringToFile = str => {
   location.href = `${downloadUrl}`;
 };
 
-export { openUrl, downloadStringToFile };
+const goToPage = page => {
```

Comment:
Here you can just use openUrl without an event.

Reply:
👍